api/src/DataTransformer/UsersOutputDataTransformer.php

```diff
@@ -31,19 +31,25 @@ public function transform($data, string $to, array $context = [])
         if(!empty($output->users2teams)){
             foreach($output->users2teams as $users2teams){
                 $team =  $users2teams->getTeams();
+                $teamUsers2Teams = $team->getUsers2teams();
```

Comment:
Look like we loop and query users for each team. Is there a better option to prevent loop queries using JOIN? 

Same issue for experiments. `$experiments = $data->getExperiments();`